eo-maven-plugin/src/main/java/org/eolang/maven/CleanFiles.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import java.io.File;
+
+/**
+ * Deleting the directory with files.
+ *
+ * @since 0.52
+ */
+final class CleanFiles {
```

Comment:
@Suban05 I think `CleanFiles` is not a really good name for the object, let's name it just `Deleted`, make it implemented an interface `Supplier` or `Scalar` and make it able to work not with directory but with file also.